packages/paste-color-contrast-utils/src/utils.ts

```diff
@@ -196,3 +207,29 @@ export const getContrastRatingsOfTokensWithUIControlContrastRequirements = (
     'uicontrol_contrast_pairing'
   );
 };
+
+/**
+ * Takes all design tokens in an object and determines which ones
+ * have a data visualization contrast ratio requirement against another token
+ * based on the annotations made in the raw design token files. Those token
+ * pairs then have the contrast ratio between them checked and rated.
+ * The result of which is returned as an array of ratings
+ *
+ * @param {Partial<GenericTokensShape>} tokens
+ * @return {*}  {TokenPairContrastRating[]}
+ */
+export const getContrastRatingsOfTokensWithDataVisualizationContrastRequirements = (
+  tokens: Partial<GenericTokensShape>
+): TokenPairContrastRating[] => {
+  // always use the Default raw JSON to get the pairings as other themes won't inherit them automatically
+  const defaultThemeRawJSON = DefaultRawTokenJSON.props;
+  const tokenWithUIControlContrastRequirements = getTokensWithDataVisualizationContrastRequirements(
```

Comment:
This maybe a copy pasta variable name? TokenWithDataVisualizationContrastRequirements?